### PR TITLE
Labs API setup

### PR DESCRIPTION
**Pull request by agordon1123**
# Description

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [X] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

- [ ] Test A
- [ ] Test B

# Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

**Comment by agordon1123** [comment]
Merging to be able to work together with Armando. The queries are a WIP. You will need to create environment variables to use the axiosLabsGraphQL function.

**Review by danbergelt — APPROVED** [review]
Looks great!

**Review by armandoroman1016 — APPROVED** [review]

File: mission-control-client/src/utils/axiosLabsGraphQL.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import axios from "axios";
+
+const axiosLabsGraphQL = axios.create({
+  baseURL: process.env.REACT_APP_LABS_API_URL,
```
Comment (dionys93) [inline mission-control-client/src/utils/axiosLabsGraphQL.js:4]: I don't think it's necessary to  have the URL in the environment, but it's not a horrible thing to have.

File: mission-control-client/src/queries/index.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
+
```
Comment (dionys93) [inline mission-control-client/src/queries/index.js:1]: Awesome

**Review by dionys93 — APPROVED** [review]

**Pull request merged**